docs/meshcloud.policies.md

```diff
@@ -0,0 +1,64 @@
+---
+id: meshcloud.policies
+title: meshPolicies
+---
+
+## Introduction: what are meshPolicies?
+
+A meshPolicy is a set of rule(s) between two [meshObjects](meshcloud.index.md#introduction) that are globally defined by your organization. Each rule describes what [tags](meshcloud.tag-schema.md) have to match on each meshObject, in order to comply with the meshPolicy. meshPolicies are enforced in various places when attaching meshObjects.
+  
+### When am I impacted by meshPolicies?
+
+> If your organization has no policies defined at all, the information below is not relevant.
+
+Note that it depends if your organization uses policies and on which meshObjects they are applied. We will discuss all possible places here.
+You may encounter policies in the meshPanel when doing any of the following actions:
+
+1. Creating a new project
+    1. When adding a meshLandingZone to a new project, all policies are evaluated between a 'meshProject' and a 'meshLandingZone'
+    2. Upon saving a new project, all policies between 'meshCustomer' and 'meshProject' are evaluated. (read use cases below to know why this is useful
+2. Editing a project
+    1. When adding a new location with a meshLandingZone all policies are evaluated between the 'meshProject' and the selected 'meshLandingZone'.
+    2. When adding a new meshCustomerUserGroup to a meshProject, all policies are evaluated between the 'meshProject' and the 'meshCustomerUserGroup'.
+    3. When changing a tag value (e.g. changing the environment) of a project, **all** policies related to the meshProject are evaluated as it impacts many meshObjects. The following meshObjects will be evaluated on to the project:
+        - the meshCustomer the project lives in
+        - all assigned meshLandingZones
+        - all assigned meshUsers
+        - all attached meshCustomerUserGroups
+3. Adding a meshCustomerUserGroup to a meshCustomer. All policies between 'meshCustomer' and 'meshCustomerUserGroup' are evaluated on the given meshCustomer and 'meshCustomerUserGroup'.
+
+### What happens when I violate a meshPolicy?
+
+It might happen, consciously or unconsciously, that you violate one or more policies. At every place in the meshPanel where this is possible, we prevent you from finalizing the violation and you will be prompted with an error message explaining which meshPolicy or policies you violated, and why.
```

Comment:
As a partner I can change tags on a LZ which results in violated policies. So we dont restrict every user from moving forward with changes that will lead to non-compliance